mysql/README.md

```diff
@@ -29,13 +29,20 @@ On each MySQL server, create a database user for the Datadog Agent.
 
 The following instructions grant the Agent permission to login from any host using `datadog@'%'`. You can restrict the `datadog` user to be allowed to login only from localhost by using `datadog@'localhost'`. See [MySQL Adding Accounts, Assigning Privileges, and Dropping Accounts][5] for more info.
 
-For MySQL versions 5.6 and 5.7, create the `datadog` user with the following command:
+For MySQL 5.6, MySQL 5.7 or MariaDB, create the `datadog` user with the following command:
```

Comment:
Please don't add documentation for MariaDB here. We don't test our agent against MariaDB, and do not guarantee that it works as we do with the official MySQL versions.